### PR TITLE
Examples updated

### PR DESCRIPTION
**Pull request by mkuculyma**
Summary of changes:
- global config example added
- task with Influxdb publisher added

### CODE REVIEW

File: examples/cfg/cfg.json [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+  "control": {
+    "plugins": {
+      "collector": {
+        "swap": {
+          "all": {
+            "proc_path": "/proc"
+          }
+        }
+      },
+      "publisher": {},
```
Comment (kindermoumoute) [inline examples/cfg/cfg.json]: here

File: examples/cfg/cfg.json [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+      "processor": {}
```
Comment (kindermoumoute) [inline examples/cfg/cfg.json]: here

File: examples/tasks/swap-influxdb.json [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+                "/intel/procfs/swap/io/in_pages_per_sec": {},
+                "/intel/procfs/swap/io/out_bytes_per_sec": {},
+                "/intel/procfs/swap/io/out_pages_per_sec": {}
+            },
+            "config": {
+                "/intel/procfs/swap": {
+                    "proc_path": "/proc"
+                }
+            },
+            "process": null,
```
Comment (kindermoumoute) [inline examples/tasks/swap-influxdb.json]: here

Comment (mkuculyma) [inline examples/tasks/swap-influxdb.json]: fixed

File: examples/cfg/cfg.json (hunk 2 above)
Comment (mkuculyma) [inline examples/cfg/cfg.json]: fixed

File: examples/cfg/cfg.json (hunk 1 above)
Comment (mkuculyma) [inline examples/cfg/cfg.json]: fixed

**Pull request merged**